packages/validator/src/commands/validateCmd.ts

```diff
@@ -0,0 +1,24 @@
+import * as utils from './utils';
+import { validate } from '../validator';
+import { Log, templates } from '../types';
+
+let template = process.env.npm_config_template || process.argv[2];
+const specs = process.env.npm_config_specs || process.argv[3];
+const version = process.env.npm_config_templateVersion || process.argv[4];
+
+const messages: Log[] = [];
+
+if (templates[template.toLowerCase() as keyof typeof templates]) {
+  template = utils.getPath(templates[template.toLowerCase() as keyof typeof templates], messages, version);
+} else if (version) {
+  messages.push({
+    level: 'warning',
+    message: 'Version argument will be ignored when validating provided template file',
+  });
```

Comment:
You're mixing using `logger` functions and writing the messages yourself. Please, stick to one.

Reply:
This place got fixed somewhere, but some parts of code are older than `logger` and even though I'm trying to replace everything with `logger` it is possible that it is still not used here and there...